includes/admin/meta-boxes/views/subscription-scheme.php

```diff
@@ -11,7 +11,10 @@
 }
```

Comment:
See https://github.com/Prospress/woocommerce-subscribe-all-the-things/pull/148

I think that `filemode` was accidentally set to `true` in one of the SATT repos I've been doing most of the work so far.

With `filemode` set to `false`, files should be 644 and directories 755 -- right?